deps/tinyxml.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "tinyxml",
+    "buildsystem": "simple",
```

Comment:
Wrong build system type. `rm-configure` and maybe a custom `script` source for `autoreconf` if needed.

Reply:
it just needs regenerating because it is old. do not understand why a custom script would be better, this works and is simple.

Reply:
It is a minor difference but a style I prefer. For example you make the mistake of not building with multiple jobs, those details aren't a problem when you let `flatpak-builder` just do its thing.

Reply:
does `flatpak-builder` know to run `autogen.sh` or `bootstrap`?

I tried both `autogen.sh` and `autogen`. And the docs at http://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html say "autogen script" but flatpak-builder fails.

Reply:
https://github.com/flatpak/flatpak-builder/commit/32c022df5b00ee38117a962055ed159aa6820f79

just needs to be `0755`, ok.